Backpack/Dialog/Classes/BPKDialogView.m

```diff
@@ -111,7 +111,7 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
 - (void)setupViews {
     self.backgroundView = [[UIView alloc] initWithFrame:CGRectZero];
     self.backgroundView.backgroundColor = self.dialogContentViewBackgroundColor;
-    self.backgroundView.layer.cornerRadius = BPKCornerRadiusSm;
+    self.backgroundView.layer.cornerRadius = BPKCornerRadiusLg;
```

Comment:
AFAIK we already have a property for determining the cornerRadius. So how does this affect that?

Reply:
That is a good question @georgegillams and it probably means this needs some more work and a further discussion with Dima.